reviewkitteh/autokitteh.yaml

```diff
@@ -0,0 +1,31 @@
+# This YAML file is a declarative manifest that describes the setup
+# of the AutoKitteh project ReviewKitteh.
+# ReviewKitteh integrates GitHub, Google Sheets and Slack.
+#
+# Before applying this file:
+# - Modify the values in the project's "vars" section
+#
+# After applying this file, initialize this AutoKitteh project's
+# GitHub, Google Sheets and Slack connections.
+
+version: v1
+
+project:
+  name: reviewkitteh
+  vars:
+    - name: "CHANNEL"
+      value: "pasha-slack-test"
+    - name: "SHEET_ID"
+      value: "16xnDhSm9JWa2nZNs65XtQ19N94KddZYAI-lovJkdDW0"
```

Comment:
```suggestion
    - name: CHANNEL
      value: pasha-slack-test
    - name: SHEET_ID
      value: 16xnDhSm9JWa2nZNs65XtQ19N94KddZYAI-lovJkdDW0
```

Reply:
That's actually not what I intended to have there. I took that out and then overwrote by accident. 

Can you look at the latest change?